src/main/scala/zio/prelude/NonEmptyTraversable.scala

```diff
@@ -85,6 +85,12 @@ trait NonEmptyTraversable[F[+_]] extends Traversable[F] {
     reduceMap(fa)(identity)
   }
 
+  /**
+   * Reduces the non-empty collection of associative elements.
+   */
+  def reduce1[A: Associative](fa: F[A]): A =
```

Comment:
I would use `reduce` for `NonEmptyTraversable` and `reduceOption` for `Traversable`.

Reply:
I can't, the name `reduce` is already taken by the method above.